proxy/proxy.go

```diff
@@ -1241,8 +1244,10 @@ func (p *Proxy) errorResponse(ctx *context, err error) {
 		code = p.defaultHTTPStatus
 	case ok && perr.err == errRatelimit:
 		code = perr.code
+	case code == 499:
+		p.log.Errorf("client canceled after %s, route %s with backend %s %s, status code %d: %v", time.Since(ctx.startServe), id, backendType, backend, code, err)
 	default:
-		p.log.Errorf("error while proxying, route %s with backend %s, status code %d: %v", id, backend, code, err)
+		p.log.Errorf("error while proxying after %s, route %s with backend %s %s, status code %d: %v", time.Since(ctx.startServe), id, backendType, backend, code, err)
```

Comment:
the first parameter should be %v because we feed it with a time.Duration value

Reply:
Nope, I want to show the string representation, see example: https://play.golang.org/p/ZQW0Pl-UnlA

Reply:
i updated the example a bit: https://play.golang.org/p/ZBwdgnJLmZh
plus, here's the doc, seek for %s: https://godoc.org/fmt
;)